### ISSUE TITLE
Could you release the corresponding binary file that used for generating /data/done

### ISSUE THREAD

**Issue by Brubbish** [yonsei-cysec/XBA#1]
Hi, I'm a bit confused about the content in the disasm.json in the subfolder in /data/done/, which I thought should be the content of  strings, function names, and disassembly codes in basic blocks. But I found that, in the files, some keys don't have corresponding value, and also some values that seem to be the source files' path which should be in the debug section(though not used in gcn-relation.json). 

Take /data/done/curl/disasm.json for an example, it has 
```
"6": [], "7": ["jmp", "ptr8", "SSL_CTX_set_srp_username"], "8": ["SSL_CTX_set_srp_username"], "9": [],
....
"77467": ["../lib/warnless.c"], "77468": ["((((("], "77469": ["memdebug.c"]
```

I wonder what information you extracted and also the data structure of the file. Also, it would be nice if you could release the binary file thus I may solve other problems if encountered.

Plus, in the README, it says that disasm.json is used for generating BOW, but in the build_vocab.py, it uses disasm_innereye.json to build vocabulary. Did I get it wrong?

**Comment by cxxz16** [comment]
yes, i also hope that...Simply providing a dataset cannot be replicated from scratch

**Comment by Brubbish** [comment]
@cxxz16 I have reimplemented the data-extracting code. our group (old g1) was quite busy recently, I may release it after the validation test, or if you interest in working on that together
